extensions/xcode/package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "xcode",
+  "title": "Xcode",
+  "version": "1.0.0",
+  "author": "Sven Tiigi",
+  "license": "MIT",
+  "description": "Open your recent Xcode Projects, view the latest Xcode Releases and manage your Xcode Simulators",
```

Comment:
You might also add mention of "other utilities" since you added commands like clear derived data, etc.